### PR TITLE
Refinance context overhaul

### PR DESCRIPTION
**Pull request by piekczyk**
# [Refinance context overhaul](https://app.shortcut.com/oazo-apps/story/15113/multiple-refinance-contexts)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- refactored refinance context so its possible to maintain multiple context states
- added refinance context to portfolio
- added landing page banner
  
## How to test 🧪
  <Please explain how to test your changes>

- nothing to test right now

### CODE REVIEW

File: features/refinance/MakerRefinanceContext.tsx
Comment (piotrwitek) [inline features/refinance/MakerRefinanceContext.tsx:1]: these files were moved could you try to use git mv to preserve the file history instead? I dunno what was changed

File: features/refinance/RefinanceContext.tsx
Comment (piotrwitek) [inline features/refinance/RefinanceContext.tsx:1]: same here

**Review by piotrwitek — CHANGES_REQUESTED** [review]

File: features/refinance/MakerRefinanceContext.tsx
Comment (piekczyk) [reply features/refinance/MakerRefinanceContext.tsx:1]: this one was actually deleted

Comment (piotrwitek) [reply features/refinance/MakerRefinanceContext.tsx:1]: To be fair, it wasn't deleted but refactored as helper

**Review by piotrwitek — APPROVED** [review]
Looks good

**Pull request merged**